src/som/interpreter/Frame.java

```diff
@@ -217,25 +217,13 @@ public void copyArgumentsFrom(final Frame frame) {
     }
   }
 
-  public void printStackTrace(final SAbstractObject nilObject) {
```

Comment:
Hm, you're already relying on an implicit stack? might still be useful for debugging.